_appmap/recorder.py

```diff
@@ -18,6 +29,8 @@ class Recorder(ABC):
     Note that the abstract methods have implementations for use by subclasses.
     """
 
+    _aborting = False
```

Comment:
Is there any chance that this will cause unintended consequences? I guess it’s so there is only one exception per recorder. 

Other recorders (eg different tests that may be running concurrently or subsequently) will continue?

Reply:
From what I've seen it's only possible to have one active `Recorder` at a time, though I don't see why this couldn't be a member variable

Reply:
The variable is shared by all instances of Recorder, so the global Recorder and request recorders all see the same value. Once one of them has a lot of events, they'll all stop adding events. 

How concurrent tests behave will depend on how they're isolated. I think we've got bigger problems than this if they share state.

Subsequent tests reset `_aborting`.

Reply:
> From what I've seen it's only possible to have one active `Recorder` at a time, though I don't see why this couldn't be a member variable

You definitely can, because you can have a global recorder and a per-thread recorder. Sharing this ensures they'll all abort once things get out of hand.

This is working for me in Django now, and since it's disabled by default, end-users won't see it. If you all would like to use soon, we can leave it as-is, and revisit it later.